crates/ef-testing/src/evm_sequencer/evm_state/v0.rs

```diff
@@ -69,7 +92,7 @@ impl Evm for KakarotSequencer {
         // Initialize the allowance storage var.
         let allowance_key_low = get_storage_var_address(
             "ERC20_allowances",
-            &[starknet_address.into(), *KAKAROT_ADDRESS.0.key()],
+            &[*starknet_address.0.key(), *KAKAROT_ADDRESS.0.key()],
```

Comment:
we should choose one way of getting kakarot address, either self.environment.kakarot_address or the lazy static, but not both

Reply:
missed that one!